omnibus-sepolia-carina.toml

```diff
@@ -0,0 +1,547 @@
+skewScale = "<%= settings.skew_scale %>"
+fundingVelocityClamp = "<%= settings.funding_velocity_clamp %>"
+minCreditPercent = "<%= settings.min_credit_percent %>"
+minMarginUsd = "<%= settings.min_margin_usd %>"
+minMarginRatio = "<%= settings.min_margin_ratio %>"
+incrementalMarginScalar = "<%= settings.incremental_margin_scalar %>"
+maintenanceMarginScalar = "<%= settings.maintenance_margin_scalar %>"
+maxInitialMarginRatio = "<%= settings.max_initial_margin_ratio %>"
+liquidationRewardPercent = "<%= settings.liquidation_reward_percent %>"
+liquidationLimitScalar = "<%= settings.liquidation_limit_scalar %>"
+liquidationWindowDuration = "<%= settings.liquidation_window_duration %>"
+liquidationMaxPd = "<%= settings.liquidation_max_pd %>"
+
+# --- Configure collateral for BFP --- #
+#
+#
+
+[invoke.set_margin_collateral_configuration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarginCollateralConfiguration"
+args = [
+    [ # collateral addresses
+        "<%= system.USDProxy.address %>",
+        "<%= settings.weth_address %>",
+        "<%= settings.wsteth_address %>",
+    ],
+    [ # oracle node ids
+        "<%= imports.system.settings.const_one_oracle_id %>",
+        "<%= settings.eth_oracle_id %>",
+        "<%= settings.wsteth_oracle_node_id %>",
+    ],
+    [ # maxAllowances
+        "<%= settings.susd_max_collateral %>",
+        "<%= settings.weth_max_collateral %>",
+        "<%= settings.wsteth_max_collateral %>",
+    ],
+    [ # skew_scales
+        "<%= settings.susd_skew_scale %>",
+        "<%= settings.weth_skew_scale %>",
+        "<%= settings.wsteth_skew_scale %>",
+    ],
+    [ # rewardDistributors
+        "0x0000000000000000000000000000000000000000",
+        "<%= settings.weth_rewards_distributor %>",
+        "<%= settings.wsteth_rewards_distributor %>",
+    ],
+]
+
+# --- Configure Core --- #
+#
+#
+
+# Add all perp market collateral as core collateral.
+# ETH and SNX have already been added from imports.
+#
+#
+
+[invoke.configure_wsteth_collateral]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureCollateral"
+
+[[invoke.configure_wsteth_collateral.args]]
+tokenAddress = "<%= settings.wsteth_address %>"
+oracleNodeId = "<%= settings.wsteth_oracle_node_id %>"
+issuanceRatioD18 = "<%= settings.wsteth_issuance_ratio_d18 %>"
+liquidationRatioD18 = "<%= settings.wsteth_liquidation_ratio_d18 %>"
+liquidationRewardD18 = "<%= settings.wsteth_liquidation_reward_d18 %>"
+minDelegationD18 = "<%= settings.wsteth_min_delegation_d18 %>"
+depositingEnabled = false
+
+# --- Configure maximum market collateral in Core --- #
+#
+#
+
+[invoke.weth_maximum_market_collateral]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "configureMaximumMarketCollateral"
+args = [
+    "<%= settings.eth_market_id %>",
+    "<%= settings.weth_address %>",
+    "<%= settings.weth_max_pool_collateral %>",
+]
+
+[invoke.wsteth_maximum_market_collateral]
```

Comment:
for core we would also need to add `configureMaximumMarketCollateralValue` that limits max market value in $

Reply:
I dont think that is true. Core does not have any maximum defined in USD and `configureMaximumMarketCollateralValue` does not exists. 

I think you might be thinking about the `setMaxMarketValue` and `setMaxMarketSize` which are not core method but perpsv3 specific methods

Reply:
You are right, I've mixed up perps v3 with this.